data_specs/Fractional_Cover_specs.ipynb

```diff
@@ -84,16 +94,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "|Specification | |\n",
-    "|----------|-------------|\n",
-    "|Product name| Fractonal Cover |\n",
-    "|Cell size - X (metres) | 30 |\n",
-    "|Cell size - Y (metres) | 30 |\n",
-    "|Coordinate reference system | Universal Transverse Mercator (UTM) |\n",
-    "|Temporal resolution | 16 days |\n",
-    "|Temporal range| 1984 &ndash; present |\n",
-    "|Parent dataset| Landsat Collection 2 Surface Reflectance |\n",
-    "|Update frequency| Daily |"
+    "|Specification | | |\n",
+    "|----------|-------------|-------------|\n",
+    "|Product name| Fractonal Cover | Fractonal Cover Annual Summary|\n",
+    "|Cell size - X (metres) | 30 | 30 |\n",
+    "|Cell size - Y (metres) | 30 | 30 |\n",
+    "|Coordinate reference system | Universal Transverse Mercator (UTM) | EPSG: 6933 |\n",
```

Comment:
```suggestion
    "|Coordinate reference system | Cylindrical Equal Area | EPSG: 6933 |\n",
```

Reply:
Are the FC scenes in Cylindrical Equal Area?

Reply:
Oh, nope. Only the summary. Scenes are still UTM.